languages/python/reference/exercise-concepts/markdown.md

```diff
@@ -0,0 +1,48 @@
+# Concepts for markdown
+
+[Example Implementation](https://github.com/exercism/python/blob/master/exercises/markdown/example.py)
+
+## General  
+
+
+- functions: This exercise requires a re-write of pre-esiting code that uses functions to parse passed-in text in _markdown_.  Tests expect a function named `parse` that can be called to transform the _markdown_ formatted text and return HTML formatted text.
```

Comment:
Spelling: "pre-esisting" -> "pre-existing". Perhaps the "re-write of pre-existing code" bit could be its own _refactor_ concept?

Reply:
Pulled `refactor` out:

> -refactor:  Reviewing and rewriting (_or re-organizing_) code for clarity and efficiency. This exercise requires a re-write of pre-existing code that uses functions to parse passed-in text in _markdown_.